lib/helpers/sendTransaction.dart

```diff
@@ -39,7 +40,18 @@ Future<bool> sendAssetTransferTransactionDialog(
   try {
     transactionKey = await qubicCmd.createAssetTransferTransaction(seed,
         destinationId, assetName, issuer, numberOfAssets, destinationTick);
-    await getIt.get<QubicLiveApi>().submitTransaction(transactionKey);
+    final transactionId =
+        await getIt.get<QubicLiveApi>().submitTransaction(transactionKey);
+    final pendingTransaction = TransactionVm(
+        id: transactionId,
+        sourceId: sourceId,
+        destId: destinationId,
+        amount: 1000000, //Fixed for asset transfer
+        status: ComputedTransactionStatus.pending.name,
```

Comment:
Actually I just saw there is a constant (but not used in the app) for this purpose
![image](https://github.com/user-attachments/assets/da776b1c-c785-4857-b019-1e5372e4ba73)

Please use it and also use it in the form
![image](https://github.com/user-attachments/assets/0d2997af-9d7d-4e30-85da-726e478703ab)